frontend/package.json

```diff
@@ -38,6 +40,7 @@
     "worker-loader": "^3.0.8"
   },
   "devDependencies": {
+    "@types/luxon": "^1.26.5",
```

Comment:
Move types to dependencies otherwise deploy will fail

Reply:
Done